src/target/move.c

```diff
@@ -341,10 +344,22 @@ void move_front_many(int cells)
 	entered_next_cell();
 }
 
+/**
+ * @brief Move front a defined distance.
+ *
+ * @param[in] distance Distance to travel.
```

Comment:
Include param[in] end_linear_speed

Reply:
4e0d4ddcb7824b6d9847b7600fc6053c53f2015f